collections/attachments.js

```diff
@@ -0,0 +1,58 @@
+Attachments = new FS.Collection("attachments", {
+    stores: [ 
+        new FS.Store.FileSystem("attachments", { path: "~/uploads/attachments" }) // XXX PATH ???? 
```

Comment:
I think the path should be something like `/var/libreboard/uploads/attachements`